.github/workflows/canister-tests.yml

```diff
@@ -252,6 +282,60 @@ jobs:
           name: e2e-screenshots-${{ matrix.device }} ${{ matrix.start-flag }}
           path: screenshots/**/*.png
 
+  dummy-auth-tests:
+    runs-on: ubuntu-latest
+    needs: docker-build-dummy-auth
+    env:
+      DFX_VERSION: 0.8.3
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Use Node.js
+        uses: actions/setup-node@v2
+        with:
+          # must use 16.x because 14.x doesn't understand how to pull repos
+          # using https (for dfx-proxy)
+          node-version: 16.x
+
+      - name: Install DFX
+        run: |
+          echo Install DFX Version: "$DFX_VERSION"
+          yes | sh -ci "$(curl -fsSL https://sdk.dfinity.org/install.sh)"
+
+      # Helps with debugging
+      - name: Show versions
+        run: |
+          echo dfx --version
+          dfx --version
+
+      - name: Start replica
+        run: |
+          dfx start --background
+
+      - name: 'Download wasm'
+        uses: actions/download-artifact@v2
+        with:
+          name: internet_identity_dummy_auth.wasm
+          path: .
+
+      - name: Deploy II
+        run: |
+          echo installing
+          shasum -a 256 ./internet_identity.wasm
+
+          dfx canister --no-wallet create --all
+          dfx canister --no-wallet install --mode install internet_identity --argument '(null)'
+
+      - name: Run tests
+        working-directory: dummy-auth-tests
+        run: |
+          npm ci
+          npm run test
+
+      - name: Stop replica
+        run: |
+          dfx stop
```

Comment:
Just out of curiosity, does the job keep running as long as dfx is running?  I would have expected the VM to be killed regardless.  This is not a change request.  Having a stop certainly doesn't harm, is safe and good.  I'm seeking more insights into how GH actions work.